src/common/menucmn.cpp

```diff
@@ -296,6 +296,14 @@ void wxMenuItemBase::SetAccel(wxAcceleratorEntry *accel)
     SetItemLabel(text);
 }
 
+void wxMenuItemBase::AddExtraAccel(wxAcceleratorEntry *accel)
+{
+    if ( accel )
```

Comment:
If we need to pass a pointer here, this should be a `wxCHECK`, as I don't see why would you ever want to intentionally pass a null pointer here. But better would be to avoid pointers in the first place.